pkg/controller/nodelifecycle/scheduler/taint_manager_test.go

```diff
@@ -130,76 +130,69 @@ func TestFilterNoExecuteTaints(t *testing.T) {
 }
 
 func TestCreatePod(t *testing.T) {
-	testCases := []struct {
-		description                   string
+	testCases := map[string]struct {
```

Comment:
Same in other places below...

Reply:
That was my call. The advantage of using map over a list is that the tests would run in random order each time. This could help the developer catch unwanted dependencies between their subtests.

Reply:
Using description as a key in a map also ensures there are no two test cases with the same description - this could happen if someone copy-pastes test cases.